tests/regression/77-lin2vareq/22-cast-to-short2.c

```diff
@@ -14,14 +16,14 @@ int main() {
   printf("unsignedtounsigned: %hu\n", unsignedtounsigned);
 
   if (unsignedtounsigned == 4294967295) {
-//    __goblint_check(0); // NOWARN (unreachable)
+    __goblint_check(0); // NOWARN (unreachable)
     return (-1);
   }
   if (allbits == 4294967295 && signedallbits == -1 && unsignedtosigned == -1 &&
-      unsignedtounsigned == 65535) {
- //   __goblint_check(1); // reachable
+    unsignedtounsigned == 65535) {
+    __goblint_check(1); // reachable
     return (-1);
   }
- // __goblint_check(0); // NOWARN (unreachable)
+
   return (0);
 }
```

Comment:
These `__goblint_checks` here seem strange. Are we sure they have hold in the concrete? If they do, we are unsound, as we claim the if is never entered.